platform/web/js/libs/library_godot_input.js

```diff
@@ -38,41 +38,57 @@ const GodotIME = {
 	$GodotIME: {
 		ime: null,
 		active: false,
+		focusTimerIntervalId: -1,
 
 		getModifiers: function (evt) {
 			return (evt.shiftKey + 0) + ((evt.altKey + 0) << 1) + ((evt.ctrlKey + 0) << 2) + ((evt.metaKey + 0) << 3);
 		},
 
 		ime_active: function (active) {
-			function focus_timer() {
-				GodotIME.active = true;
-				GodotIME.ime.focus();
+			function clearFocusTimerInterval() {
+				clearInterval(GodotIME.focusTimerIntervalId);
+				GodotIME.focusTimerIntervalId = -1;
 			}
 
-			if (GodotIME.ime) {
-				if (active) {
-					GodotIME.ime.style.display = 'block';
-					setInterval(focus_timer, 100);
-				} else {
-					GodotIME.ime.style.display = 'none';
-					GodotConfig.canvas.focus();
-					GodotIME.active = false;
+			function focusTimer() {
+				if (GodotIME.ime == null) {
```

Comment:
You can probably just check for `!GodotIME.ime` since all of the falsy values can't call focus anyway

Reply:
I will not. Because it's not a check that we expect to do per se.